pagopa/pn-frontend
# PN-6902 - show analog failure workflow

## Short description
Add a title and copy to the ANALOG_FAILURE_WORKFLOW timeline event category, so that the events of this category are explicitly shown in the FE timeline.

## List of changes proposed in this pull request
- Added the needed `AnalogFailureWorkflowStep` and integrated it into the `TimelineStepFactory`.
- Included ANALOG_FAILURE_WORKFLOW into the list of allowed event categories.
- Provided the corresponding title and mono/multirecipient versions of the copy.

## How to test
**NB**  
Haven't tested for PG, only PA as sender and PF as recipient.

Enter the details of a notification including a ANALOG_FAILURE_WORKFLOW timeline event. The event should be explicit in timeline with this copy
```
Il destinatario è risultato irreperibile assoluto. Si deposita e mette a disposizione l'avviso di avvenuta ricezione.
```
and similar for PA access to multirecipient notifications. This event should be shown both in sender and recipient access to the notification.

Examples in TEST
- single recipient: REMV-TXLH-KWTP-202306-M-1 / Comune di Palermo (suitable user grossini) / leonardo
- multirecipient: YTJQ-TWTW-NLWH-202306-K-1 Comune di Sappada (suitable user grossini) / leonardo
